fuzzers/int_maketodo.py

```diff
@@ -86,11 +96,70 @@ def maketodo(
             include = re.match(exclude_re, line) is None
 
         if include:
-            print(line)
+            filtered_todos.add(line)
         else:
             drops += 1
         lines += 1
     verbose and print('Print %u entries w/ %u drops' % (lines, drops))
+    '''
```

Comment:
This should still be a comment rather than a string.

Reply:
I wanted to be consistent with the rest of the comments in the int_maketodo.py file. These other comments also tend to use multi line strings for commenting.
Should I change it to '#' anyway?

Reply:
They are docstrings - see https://www.python.org/dev/peps/pep-0257/

Reply:
Changed it to multi-line docstrings then.

Reply:
A docstring goes on a function, not in the middle of a function.
```python
def abc(xxx):
    ''''This is a doc string''''
    pass

def abc(xxx):
    """This is a doc string"""
    pass

def abc(xxx):
    a = 1

    """This is a *not* doc string"""

   y = 1
```

Reply:
This actually made me think that it would be cleaner to extract the balancing code into a function and then use a doc string instead of a long comment.

Reply:
That is a good idea.

Reply:
Done